lib/datapacksexpand.js

```diff
@@ -962,6 +964,38 @@ DataPacksExpand.prototype.processDataPackData = function(dataPackType, parentNam
     }
 };
 
+DataPacksExpand.prototype.addDataPackToFullManifest = function(extension, dataPackType, expansionData) {
```

Comment:
This method should be split into components of what it does. Get referenced datapacks by scss. Then an add to jobfile method should be in datapacksjob.js or datapacksutils.js.